astrodb_utils/utils.py

```diff
@@ -173,20 +162,27 @@ def find_source_in_db(
     if len(db_name_matches) == 0 and coords:
         location = SkyCoord(ra, dec, frame="icrs", unit="deg")
         radius = u.Quantity(search_radius, unit="arcsec")
-        logger.info(
-            f"{source}: No Simbad match, trying coord search around"
+        logger.debug(
+            f"{source}: No Simbad match, trying coord search around "
             f"{location.ra.degree}, {location.dec}"
         )
         db_name_matches = db.query_region(
             location, radius=radius, ra_col=ra_col_name, dec_col=dec_col_name
         )
 
     # If still no matches, try to get the coords from SIMBAD
-    if len(db_name_matches) == 0:
+    if len(db_name_matches) == 0 and use_simbad:
         simbad_result_table = Simbad.query_object(source)
         if simbad_result_table is not None and len(simbad_result_table) == 1:
+            logger.debug(f"simbad colnames: {simbad_result_table.colnames} \n simbad results \n {simbad_result_table}")
+            if 'RA' in simbad_result_table.colnames:
```

Comment:
There's probably a way to convert the column names to force them to be lower-case, such as `simbad_result_table.colnames = [x.lower() for x in simbad_result_table.colnames]` but this works fine for now.

Reply:
Let's leave as is. I don't want to change the column names.